src/main/resources/add-ons-to-index.json

```diff
@@ -4471,6 +4471,23 @@
         "package": "emrmonitor"
       }
     },
+    {
+      "uid": "org.openmrs.omod.facelist",
+      "type": "OMOD",
+      "name": "FACE LIST MODULE",
```

Comment:
Can you change the format of the name to "Face List Module" instead of all capitals as well, please?